build/parent/pom.xml

```diff
@@ -44,7 +44,7 @@
         <eclipse-xml-search-site>https://dl.bintray.com/gamerson/eclipse-wtp-xml-search/</eclipse-xml-search-site>
         <tern-site>http://oss.opensagres.fr/tern.repository/1.2.1/</tern-site>
         <bndtools-site>https://bndtools.ci.cloudbees.com/job/bndtools.master/lastSuccessfulBuild/artifact/build/generated/p2/</bndtools-site>
-        <gradle-site>http://download.eclipse.org/buildship/updates/e46/releases/1.0/1.0.21.v20161010-1640</gradle-site>
+        <gradle-site>http://download.eclipse.org/buildship/updates/e46/snapshots/2.x/</gradle-site>
```

Comment:
Lets make it a specific version, not, 2.x, and lets do it from the 'releases' not 'snapshots'